Source/Orts.Simulation/Simulation/RollingStocks/MSTSWagon.cs

```diff
@@ -1202,7 +1221,8 @@ public virtual void Parse(string lowercasetoken, STFReader stf)
                     break;
                 case "wagon(inside": HasInsideView = true; ParseWagonInside(stf); break;
                 case "wagon(orts3dcab": Parse3DCab(stf); break;
-                case "wagon(numwheels": NumWheelsBrakingFactor = stf.ReadFloatBlock(STFReader.UNITS.None, 4.0f); break;
+                case "wagon(numwheels": MSTSWagonNumWheels= stf.ReadFloatBlock(STFReader.UNITS.None, 4.0f); break;
+                case "wagon(ortsnumberaxles": WagonNumAxles = stf.ReadFloatBlock(STFReader.UNITS.None, null); break;
```

Comment:
Also use an `int` to represent WagonNumAxles.

Reply:
Converted to Integer